### PR TITLE
Adding redirect template

### PR DESCRIPTION
**Pull request by line47**
Live preview of redirect branch - https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/redirect-config/

### Screenshot

screen shot 2018-01-24 at 5 18 27 pm

### CODE REVIEW

File: config/templates.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
     repo: 'federalist-report-template',
     branch: 'master',
     example: 'https://federalist-report-template.18f.gov',
-    title: 'Basic Report Template',
+    title: 'Basic report template',
```
Comment (line47) [inline config/templates.js]: Also sentence cased the template titles

**Comment by wslack** [comment]
should we change the grid to a 3 max to avoid the orphan effect?

**Comment by line47** [comment]
I was thinking something similar @wslack. 
I have also been wondering about stacking the template offerings like the rough screenshot below.

### Stacked template list
screen shot 2018-01-24 at 8 20 52 pm

**Comment by line47** [comment]
Hey @jseppi one thing I noticed when I was creating a new site locally and using the `redirect-config` is was not creating the new site on a `master` branch but rather keeping the `redirect-config` branch name. I thought this was fixed with issue https://github.com/18F/federalist/issues/1386

**Comment by jseppi** [comment]
Huh, yeah, that seems to be a bug in the web app. I just confirmed and will open an issue

**Comment by jseppi** [comment]
Opened https://github.com/18F/federalist/issues/1547

**Comment by jseppi** [comment]
Using a non-master branch for a template is blocked by the issue described in #1547.

**Comment by jseppi** [comment]
This should be unblocked now (thanks, @el-mapache).
@line47 - want to give it a try on your local instance after you merge `staging` into this PR?

**Comment by line47** [comment]
I confirmed that using the redirect template works locally 👍 

**Screenshots**
screen shot 2018-01-31 at 10 26 00 am
screen shot 2018-01-31 at 10 26 09 am

**Comment by line47** [comment]
We still need to address how adding a 5th template creates an orphan on a new line
@ericronne have thoughts on that? (see screenshots earlier in the thread)


CC @wslack

**Comment by ericronne** [comment]
Let me take a look @line47  …

**Comment by ericronne** [comment]
This might work better for now …
templates-tweak

**Comment by ericronne** [comment]
Note two text changes, recommending …
- Remove "template" from the template titles (since we already have that in the section title and the button)
- Change label to "View sample" instead of "View template"

**Comment by line47** [comment]
I like these ideas @ericronne 👍

**Comment by wslack** [comment]
This dramatically raises the vertical size of the page; we'd likely need something at the top to make clear of the "add your own repo" option (or flip those sections).

**Comment by ericronne** [comment]
@wslack We could flop the order and fold the git form into an accordion (and shrink the template thumbs, for overall "digestibility"), so both options are clearly presented to users …

templates-tweak-v2

**Comment by jseppi** [comment]
That looks great

**Comment by wslack** [comment]
I like the recent screenshot, but I wonder if we can make the thumbs a little larger, give them a border, and provider a little more prose for each ( @davidewarren can probably help with the prose).

I also think the "add your own" call to action is small enough that we shouldn't make folks unfurl it.

**Comment by line47** [comment]
I agree in showing the full "add your own GitHub repository" form and think it could be even simplified a bit more.

I would hesitate in making the thumbs larger if we want to try and control the length of this page also we have a preview link right there where they can see the a live demo.

I think adding a bit more descriptive language to each template would be great too!

**Comment by wslack** [comment]
I would be happy to make the thumbs smaller if we also remove the gray box - I just didn't like the vertical space being wasted.

**Comment by jseppi** [comment]
Are we considering this blocked by the redesign of the page? If so, let's create an issue for that design implementation and mark it as a dependency to this one.

**Comment by jseppi** [comment]
Sorry, I didn't just mean the actual design work, but also the implementation of that design. This seems to have drifted fairly far from just adding a new template, so it might be good to track that work separately.

**Comment by ericronne** [comment]
i defer to your wisdom 🙏 

revision...

templates-tweak-v3

**Comment by jseppi** [comment]
Filed https://github.com/18F/federalist/issues/1585

**Comment by line47** [comment]
Just a heads up, I'm going to open a new PR for the UI work on the add new site page

**Pull request merged**